### PR TITLE
Filter out storybooks in CoreGui

### LINKED ISSUE #218: Complains about CoreGui
## Describe the bug

it is giving warnings about coregui

## Expected behavior

not give warnings about coregui

## Steps to reproduce

1. install plugin

## Error message, screenshots, and extra context
RobloxStudioBeta_2023-07-09_10-24-03

also, the plugin doesn't work at all for me. not sure if it's related.

### PR DESCRIPTION
**Pull request by vocksel**
# Problem
When opening flipbook several warnings appear in the output from it attempting to load Roblox-created storybooks in CoreGui
```
  12:37:42.545  Failed to load storybook CoreGui.RobloxGui.Modules.VoiceChatPrompt.VoiceChatPrompt.storybook. Error: table expected, got nil  -  Edit - useStorybooks:40
  12:37:42.545  Failed to load storybook CoreGui.RobloxGui.Modules.Settings.Settings.storybook. Error: table expected, got nil  -  Edit - useStorybooks:40
  12:37:42.545  Failed to load storybook CoreGui.RobloxGui.Modules.InGameChat.BubbleChat.BubbleChat.storybook. Error: table expected, got nil  -  Edit - useStorybooks:40
  12:37:42.545  Failed to load storybook CoreGui.RobloxGui.Modules.TopBar.TopBar.storybook. Error: table expected, got nil  -  Edit - useStorybooks:40
  12:37:42.545  Failed to load storybook CoreGui.RobloxGui.Modules.ContactList.ContactList.storybook. Error: table expected, got nil  -  Edit - useStorybooks:40
```

# Solution

To fix this, I've updated `isStorybookModule` to return `false` for any storybook in CoreGui

Closes #218 

# Checklist

- [x] Ran `./bin/test.sh` locally before merging

### CODE REVIEW

File: src/Story/isStorybookModule.lua
Comment (passhley) [inline src/Story/isStorybookModule.lua:1]: nitpick, but could we move Services to the top of the file?

File: wally.toml
Comment (passhley) [inline wally.toml:1]: This is the same as my comment on [219](https://github.com/flipbook-labs/flipbook/pull/219)

**Review by passhley — CHANGES_REQUESTED** [review]
Overall looks fine to me, just a few changes

File: src/Story/isStorybookModule.lua
Comment (vocksel) [reply src/Story/isStorybookModule.lua:1]: Usually I follow the order of:
1. Variable to root of project
2. Services
3. Imports
4. Constants
5. Variables/functions/etc.
I'm pretty sure this is consistent across most of the codebase

**Review by passhley — APPROVED** [review]

**Pull request merged**